src/LinuxEvent.Tests/EventParseTests.cs

```diff
@@ -17,7 +17,7 @@ private void DoStackTraceTest(string path, bool doBlockedTime, List<List<string>
 
             for (int i = 0; i < stackSource.SampleIndexLimit; i++)
             {
-                var sample = stackSource.GetSampleByIndex((StackSourceSampleIndex)i);
+                var sample = stackSource.GetLinuxPerfScriptSampleByIndex((StackSourceSampleIndex)i);
```

Comment:
If we are going to add a new API here, let's modify the test to run twice and test both of these APIs.  It's important that both work.

Reply:
Sure, makes sense